src/components/sidebar/sidebar.jsx

```diff
@@ -29,9 +31,8 @@ const getLinkProps = ({ isCurrent }) => {
   return {
     className: `${
       style.link
-    } text-primary-muted dark:text-primary-muted-inverted ${
-      isCurrent ? style.linkActive : ''
-    }`,
+    } text-primary-muted dark:text-primary-muted-inverted 
+    ${isCurrent ? `${style.linkActive} ${ACTIVE_LINK_SELECTOR}` : ''}`,
```

Comment:
This is another solution @JosephusPaye

Reply:
It's so weird that the CSS module import is generating a class name with `+` in it.

Anyway, I agree that using a different selector for the active link is the way to go. In cases where I'm adding a selector solely for JS purposes, I like to use a data attribute ([some of the reasons covered here](https://pjmcdermott92.medium.com/why-i-use-custom-data-attributes-for-selecting-elements-in-javascript-58166fe074a8)).

With that approach, `getLinkProps()` will look like:

```js
const getLinkProps = ({ isCurrent }) => {
  return {
    'className': `${
      style.link
    } text-primary-muted dark:text-primary-muted-inverted
    ${isCurrent ? style.linkActive : ''}`,
    'data-active-link': isCurrent ? true : null,
  }
}
```

And the selector will look like:

```js
const activeLink = sidebarRef.current.querySelector('[data-active-link]')
```

Reply:
Makes sense! Thanks :)